datacube/storage/storage.py

```diff
@@ -61,7 +61,10 @@ def _rasterio_resampling_method(resampling):
 
 if str(rasterio.__version__) >= '0.36.0':
     def _rasterio_crs_wkt(src):
-        return str(src.crs.wkt)
+        if src.crs:
+            return str(src.crs.wkt)
```

Comment:
From the rasterio release notes:
> - The `crs` property of a dataset is now `None` instead of `CRS()` when the dataset's coordinate reference system is undefined (mapbox/rasterio#1057).

See mapbox/rasterio#1081 for more details.